app/views/course/discussion/topics/_discussion_topic_programming_file_annotation.jbuilder

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+topic = file_annotation.acting_as
+answer = file_annotation.file.answer
+question = answer.question
+submission = answer.submission
+assessment = submission.assessment
+question_assessment = assessment.question_assessments.find_by!(question: question)
+
+json.id topic.id
+json.title "#{assessment.title}: #{question_assessment.display_title}"
+json.creator submission.creator
+json.submission submission
```

Comment:
Why are we passing the whole submission? I think may be unnecessary but let me know